controllers/network_test.go

```diff
@@ -40,6 +40,8 @@ func TestMain(m *testing.M) {
 			logger.Log(3, "received node update", update.Action)
 		}
 	}()
+	os.Exit(m.Run())
```

Comment:
network tests should not rely on nodes, there is no point in this main

Reply:
a TestMain is required in every package to prevent tests from deadlocking

Reply:
```
mkasun@endeavour ~/netmaker1 (GRA-1217-tests)> grep -R TestMain *
controllers/network_test.go:func TestMain(m *testing.M) {
functions/helpers_test.go:func TestMain(m *testing.M) {
logic/pro/networkuser_test.go:func TestMain(m *testing.M) {
logic/host_test.go:func TestMain(m *testing.M) {
test/api_test.go.bak:func TestMain(m *testing.M) {
```
```
mkasun@endeavour ~/netmaker1 (GRA-1217-tests)> grep -R m.Run  *
controllers/network_test.go:    os.Exit(m.Run())
functions/helpers_test.go:      os.Exit(m.Run())
logic/pro/networkuser_test.go:  os.Exit(m.Run())
logic/host_test.go:     os.Exit(m.Run())
grep: netclient/ncutils/windowsdaemon/winsw.exe: binary file matches
test/api_test.go.bak:   os.Exit(m.Run())
mkasun@endeavour ~/netmaker1 (GRA-1217-tests)> 

```